plugin/src/main/java/com/denizenscript/denizen/objects/properties/entity/EntityMaxTemper.java

```diff
@@ -0,0 +1,53 @@
+package com.denizenscript.denizen.objects.properties.entity;
+
+import com.denizenscript.denizen.objects.EntityTag;
+import com.denizenscript.denizencore.objects.Mechanism;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import org.bukkit.entity.AbstractHorse;
+
+public class EntityMaxTemper extends EntityProperty<ElementTag> {
+
+    // <--[property]
+    // @object EntityTag
+    // @name max_temper
+    // @input ElementTag(Number)
+    // @description
+    // Controls the upper-bound for a horse-type entity's chance to be tamed.
+    //
+    // When a player mounts an entity, a number between 0 and the entity's max temper is generated.
+    // If this number is greater than or equal to the entity's current temper, the entity bucks the player, and its temper is increased by 5.
```

Comment:
this is still just backwards

Reply:
or... wonky af, idek. Point is, you had this point right the first time, i don't know why you changed it